config/routes.rb

```diff
@@ -3,8 +3,13 @@
 
   root to: "welcome#index"
 
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
+
   namespace :merchant do
     get "/", to: "dashboard#index", as: :user
+    get "/orders/:id", to: "dashboard#show", as: :orders_show
```

Comment:
Is this for showing a single order (like the link off of the dashboard)? If so, maybe :order_show might be more appropriate?

Reply:
This one is for all the pending orders, so that's why I made it plural. Happy to switch it to order if that's better!